otter/convergence/errors.py

```diff
@@ -86,3 +95,6 @@ def String(string):
         return {'string': string}
 
     Structured = identity
+
+    def UserMessage(message):
```

Comment:
It basically just gives us a convenient way to specify a user-presentable message from ad-hoc planning code. I use it in the change in planning.py.